oarepo_oidc_einfra/ext.py

```diff
@@ -38,11 +56,9 @@ def init_config(self, app):
             if k.startswith("EINFRA_"):
                 app.config.setdefault(k, getattr(config, k))
 
-    def register_sync_component_to_community_service(self, app):
-        """Registers a component to the community service.
-
-        This component is responsible for synchronizing the community to the E-INFRA Perun.
-        """
+    def register_sync_component_to_community_service(self, app: Flask) -> None:
+        """Register components to the community service."""
+        # Community -> AAI synchronization service component
         communities_components = app.config.get("COMMUNITIES_SERVICE_COMPONENTS", None)
         if isinstance(communities_components, list):
```

Comment:
Only when `EINFRA_COMMUNITY_SYNCHRONIZATION`?

Reply:
Can not do it here, because the EINFRA_COMMUNITY_SYNCHRONIZATION might not be set yet (if set from another library, order of initiation is not guaranteed)